addons/web_editor/tools.py

```diff
@@ -144,5 +151,39 @@ def get_video_thumbnail(video_url):
         response = requests.get(f'https://www.instagram.com/p/{video_id}/media/?size=t', timeout=10)
 
     if response and response.ok:
-        return image_process(base64.b64encode(response.content))
+        if resize:
+            return image_process(base64.b64encode(response.content), size=(210, 120), crop=True)
+        else:
+            return image_process(base64.b64encode(response.content))
```

Comment:
Maybe you could do that in one line with only one `return` and if not resize `size = (0, 0)` and `crop = None` ...